<launch>
    <group ns = 'Movement'>
        <node pkg="lebot" type="myControllerInput.py" name="ControllerToRos" />
        <node pkg="lebot" type="theWheel.py" name="RosToWheel"/>
        <param name="basket_color" type="string" value="blue"/>
        <param name="lebot_rate" type="int" value="27"/>
    </group>
    <include file="$(find ds4_driver)/launch/ds4_driver.launch">
        <arg name="use_standard_msgs" value="true" />
    </include>
</launch>
